### PR TITLE
Enhancement to Readme file

### PR DESCRIPTION
**Pull request by MrHerbalizer**
# Pull Request

## Related Issue:

Closes #XXXXX

### CODE REVIEW

**Review by suitupstan — CHANGES_REQUESTED** [review]
Thank you! Looks pretty cool, but can you please make the changes suggested by David?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by MrHerbalizer** [comment]
You're welcome Krish. I have amended the changes in [CONTRIBUTING.md](https://github.com/MrHerbalizer/developer-rubric-discord-bot/blob/main/CONTRIBUTING.md) as suggested by David. Have completely removed the 'Code of Conduct' section and added the link to [CODE_OF_CONDUCT.md](https://github.com/MrHerbalizer/developer-rubric-discord-bot/blob/main/CODE_OF_CONDUCT.md)

File: README.md [hunk 1]
```diff
@@ -1,3 +1,66 @@
...
+```
+### Running the bot:
+```
+npm run start
+```
+### Running the bot in development mode, with file-reloading:
+```
+npm run start:dev
+```
+$~$
```
Comment (krshkodes) [inline README.md:49]: Wondering what's this?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
-# Contributing Guidelines
+# Contributing
 
-Add specific contributing guidelines to this file as needed for the individual project.
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
 
-Questions about contributing? [Join our Discord community](https://discord.gg/Byjr6rdBUZ)
```
Comment (Panquesito7) [inline CONTRIBUTING.md:5]: I think we should keep this line. It encourages our users to join our Discord community, and we want to grow our Discord community more and more.


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (MrHerbalizer) [reply CONTRIBUTING.md:5]: I was planning to put few links, including this one, in readme.md file. But yes, we should keep it here too. Will do the needful.

**Comment by MrHerbalizer** [comment]
Hello Moderators. Thank you for reviewing the pull requests and helping me get better.
Hope we meet in future PRs...:)

**Pull request closed without merging**